vault/dendron.topic.traits.api.md

```diff
@@ -0,0 +1,82 @@
+---
+id: oOIEFJc6DTgS71mmh89WQ
+title: API
+desc: ''
+updated: 1638860903879
+created: 1638844803983
+---
+
+This contains API documentation for trait behavior. 
+
+> ❗️ Note: This is an experimental API that may have breaking changes in future iterations. An increased API surface exposing more types of functionality for traits are planned in future iterations.
```

Comment:
can you add this to `#stage.germ` and add a note reference to it in here?

Reply:
I'll be making this change within the changelog update for early seed

Reply:
See https://github.com/dendronhq/dendron-site/pull/317